use-cases/inventory.md

```diff
@@ -1,30 +1,257 @@
 # Inventory management
 
-> **Template**: This documentation page is awaiting content. Please check back later for full details.
+This use case is aimed at external Inventory Management Systems, and it describes how to use the __Mews POS API__ to managing inventory across multiple point-of-sale locations or outlets.
 
-\< Insert here a summary of the typical client and their use case for using the API \>
+## Contents
 
-## Initial configuration
+* [Product synchronization](#product-synchronization)
+* [Fetching sales data](#fetching-sales-data)
+* [Fetching the location or outlet](#fetching-the-location-or-outlet)
+* [Frequently Asked Questions](#frequently-asked-questions)
+* [Additional help](#additional-help)
 
-\< Insert content here \>
+## Product synchronization
 
-\< Reference endpoints or operations />
+In order to synchronize products with Mews POS, use the [Get products](../operations/products.md#get-products) endpoint `GET /api/v2/products`. You can call this API operation periodically to refresh the product catalogue and make sure it is in sync with your system. The product attributes in the response include name, description, [SKU](https://en.wikipedia.org/wiki/Stock_keeping_unit), barcode, and price information.
 
-> **Editable History Window**: For a full description and links to further information, see the [Mews Glossary for Open API users](https://help.mews.com/s/article/Mews-Glossary-for-Open-API-users?language=en_US).
+#### Example request:
 
-## Periodic update
+```
+GET [PlatformAddress]/api/v2/products
+```
 
-\< Insert content here \>
+Further detail can be obtained via linked entities such as `productType` (e.g. food or beverage), `modifierSet` (product qualifiers, such as pizza toppings) and `productVariant`. These can be included in the response by using the same request but specifying the relevant include query parameters – see [Essential guide > Relationships](../essential-guide/relationships.md).
```

Comment:
Maybe make modifierSet and productVariant to be in plural (to match the exact relationship name).